050_ci_L81/050_ci_L81_unsafe.py

```
@@ -0,0 +1,311 @@
+    def __init__(self):
+        self.config_source = SUBMODULES_CONFIG
+
+        self.config_override = self.read_config_override()
+        self.config = self.read_config()
+
+        self.merge_configs()
+        self.validate_config()
+
+    def read_config_override(self):
+        with open(YAML_CONFIG_OVERRIDE, 'r') as f:
+            return yaml.load(f, Loader=yaml.FullLoader)
```

Ruff (0.14.2)

Probable use of unsafe loader `FullLoader` with `yaml.load`. Allows instantiation of arbitrary objects. Consider `yaml.safe_load`.

(S506)